src/status.rs

```diff
@@ -291,7 +291,7 @@ impl<'de> Deserialize<'de> for Status {
                 impl<'de> Visitor<'de> for FieldVisitor {
                     type Value = Field;
 
-                    fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+                    fn expecting(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
I'd actually remove those lifetimes. Cargo fix adds them, but I think it actually makes the code harder to read and it works without them (same thing below...).

Reply:
What is the reasoning those get added?

Reply:
Bad heuristic (the docs for --edition-idioms mentions that it can generate broken code).

If you have named lifetimes, you can now elide the name and Rust can derive it. But if there was no explicit named lifetime before, it doesn't make sense to add an explicit unnamed lifetime.

Reply:
Fixed.